src/Database/Query/CacheableQueryBuilder.php

```diff
@@ -81,6 +86,7 @@ public function __construct(
         $this->modelIdentifier = $cacheableProperties['identifier'] ?? null;
         $this->ttl = $cacheableProperties['ttl'] ?? null;
         $this->prefix = $cacheableProperties['prefix'] ?? null;
+        $this->logChannel = Arr::get($cacheableProperties, 'logging.channel', null);
```

Comment:
Hi there, you could add `Log::getDefaultDriver()` as default value here, to have it always set.